restructure quota mgmt docs v3 (#5141)
getsentry/sentry-docs: src/docs/product/accounts/quotas/index.mdx

</Note>

## 4. Rate Limits

> If the error event rate limit for a project has been exceeded, and your subscription allows, the event will not be counted. Learn more in [Rate Limiting](/product/accounts/quotas/manage-event-stream-guide/#6-rate-limiting) or in [Error Event Limits](/product/accounts/quotas/#error-event-limits). This does not apply to transactions or attachments.

Author: ```suggestion
If the error event rate limit for a project has been exceeded, and your subscription allows, the error event or attachment will not be counted. This does not apply to transactions.
```

If you have enabled the storage of crash reports, you may set limits for the maximum number of crash reports that will be stored per issue. To set up these limits, use the slider in the "Store Native Crash Reports" option in your organization's "Security & Privacy" settings.

> ## 5. Delete, & Discard Issues

Author: ```suggestion
## 5. Delete & Discard Issues
```

![Delete and Discard](manage-event-stream-06.png)

You can find a list of these issues in the "Discarded Issues" tab in **[Project] > Settings > Inbound Filters**. From here, you can un-discarded any of these issues to receive future events.

![List of Discarded Issues](manage-event-stream-07.png)

<Note>

Once you've identified a set of discarded issues, it might make sense to go back to your SDK configuration and add the related errors into your `before-send` client-side filtering.

</Note>

### Event Repetition

Event repetition only applies to error events, not transactions or attachments. The following rules apply for event repetition and your quota:

- If you have chosen to "Delete & Discard" an issue, then _future_ error events with the same fingerprint do not count toward your quota.
- If you previously resolved an issue, a new error event counts toward your quota because it may represent a regression in your code.

> - If you have chosen to ignore alerts about error events with the same fingerprint, a new event counts toward your quota because the event is still occurring. Learn more in [Inbound Filters](#inbound-data-filters).

Author: ```suggestion
- If you have chosen to ignore alerts about error events with the same fingerprint, a new event counts toward your quota because the event is still occurring. Learn more in [Inbound Filters](#6-inbound-filters).
```

## 6. Inbound Filters

> If any inbound filter is set for a type of event or attachment, and your subscription allows, the event or attachment won't be counted. Learn more in [Inbound Data Filters](#inbound-data-filters).

Author: ```suggestion
If any inbound filter is set for a type of event or attachment, and your subscription allows, the event or attachment won't be counted. 
```
